wiki/BioSQL.md

```diff
@@ -92,8 +107,7 @@ mysqladmin -u root create bioseqdb
 ```
 
 We can then tell MySQL to load the BioSQL scheme we downloaded above.
-Change to the scripts subdirectory from the unzipped BioSQL download,
-then:
+Change to the **sql** subdirectory (see above) and then:
```

Comment:
Use the double back-ticks for directory names, ``sql``, rather than the double asterisk for bold.